src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -995,3 +995,19 @@ bool clang_cpp_convertert::get_decl_ref(
 
   return false;
 }
+
+void clang_cpp_convertert::get_default_symbol(
+  symbolt &symbol,
+  irep_idt module_name,
+  typet type,
+  irep_idt name,
+  irep_idt id,
+  locationt location)
+{
+  symbol.mode = "C++";
```

Comment:
can you make a new constexpr member called `mode` that initializes to the proper string when creating the frontend?

Reply:
Done in commit d1c213c01f64dcb3e961004d2b089f6b9d4b32d8